### DISCUSSION TITLE
Blocking Issues.

### DISCUSSION THREAD

**Discussion by the1Riddle** [uBlockOrigin/uAssets#21717] [General]
I am currently utilizing the extension on my web browser, specifically chrome. However, sometimes i get some ADs . I am curious if anyone else has encountered this issue?

**Comment by Yuki2718** [comment]
No blocker is perfect. If you see unblocked ads, report that with the built-in report tool. This is how the current blockers improve.

![reporter](https://github.com/uBlockOrigin/uBlock-issues/assets/58900598/b959c45a-69c4-4fc8-9208-71b8b02b2e82)

**Comment by u-RraaLL** [comment]
On browser start, Chromium browsers don't wait for extensions to load before loading websites. So if you "restore tabs"/keep them open or launch the browser by clicking a link somewhere else, Chrome will launch the website before uBO is ready.

You can try this setting, but it's not guaranteed to always work on Chromium browsers: https://github.com/gorhill/uBlock/wiki/Dashboard%3A-Filter-lists#suspend-network-activity-until-all-filter-lists-are-loaded

**Reply by the1Riddle** [reply]
i guess that might be the case,  let me check the link out if it might work for me.

**Comment by ItsProfessional** [comment]
Provide us the URL on which you are getting ads.
